tools/crm_attribute.c

```diff
@@ -240,6 +260,17 @@ static GOptionEntry addl_entries[] = {
       "SECTION"
     },
 
+    { "wait", 'W', 0, G_OPTION_ARG_CALLBACK, wait_cb,
+      "Wait for some event to occur before returning.  Values are 'no' (wait\n"
+      INDENT "only for the attribute daemon to acknowledge the request),\n"
+      INDENT "'local' (wait until the change has propagated to where a local\n"
+      INDENT "query will return the request value, or the value set by a\n"
+      INDENT "later request), or 'cluster' (wait until the change has propagated\n"
+      INDENT "to where a query anywhere on the cluster will return the requested\n"
+      INDENT "value, or the value set by a later request).  Default is 'no'.\n"
+      INDENT "(with -D or -u)",
```

Comment:
with node and D or u (without node, crm_attribute updates a cluster property in the CIB)

Reply:
Fixed.